base/multi.jl

```diff
@@ -980,6 +980,7 @@ function start_worker(out::IO)
     print(out, LPROC.bind_addr)
     print(out, '\n')
     flush(out)
+    ccall((:sync, "libc"), Void, ())   # necessary for SGE, which redirects and buffers stdout
```

Comment:
I wonder if the `flush` function shouldn't do this automatically. In my experience you either don't care about buffering, in which case you wouldn't all `flush` anyway, or you do and you want to flush everything as thoroughly as possible.